ovn-org/ovn-kubernetes
# Import ginkgo and gomega using pacagename on `pkg/util` and `pkg/ovn`

**-What this PR does and why is it needed **

this PR referring to #1801.

this PR import the `gomega` and `ginkgo` using the package name instead of using the "." import .
**- Special notes for reviewers**

the files changed are the `*_test.go` on the directories`pkg/ovn/*` and `pkg/util/*`.

**- How to verify it**

try to build and run the tests on the directories `pkg/ovn/*` and `pkg/util`.

to check  `pkg/util/*` run from repository root directory:
```
cd go-controller/pkg/util
go test
```
and make sure the test is successfully build.
to check  `pkg/ovn/*` run from repository root directory:
```
cd go-controller/pkg/ovn
go test
```
and make sure the test is successfully build.
**- Description for the changelog**

fixing #1801 on the directories`pkg/ovn/*` and `pkg/util/*`
